### ISSUE TITLE
Time based conditions

### ISSUE THREAD

**Issue by FadeFx** [renyuneyun/Easer#178]
So there are time based events that have an exact time to trigger, and then there are conditions also time based, but imo these should have start and end time to make any sense, otherwise we would better use an event and a dynamic (if i got this concept right its just like a variable or array) instead.

**Comment by renyuneyun** [comment]
So I guess your are suggesting "from xxx on" and "until xxx" (where "xxx" is a time) right?
This can be achieved by using chained Scripts (sorry for the naming, see #163 ), one with "after" and one with "before" (or "reverse" of "after", if no "before" is provided [yet]).

**Comment by FadeFx** [comment]
Yeah, but shouldnt that be what conditions are meant for? Imo most stuff that qualifies for a condition can be defined by an entry and an exit event i guess...

**Comment by renyuneyun** [comment]
I see your point, but this will go infinite if implemented in every Condition.

What I did is to try to extract the basic components of every Event/Condition, and leave the decision of "composition" to the user when defining Scripts. I even moved the "reversed" setting to Script but not in each Event or Condition, also for this purpose.
